docs/index.md

```diff
@@ -229,6 +229,96 @@ resource "azurerm_managed_disk" "sample" {
 }
 ```
 
+If you prefer combined multiple configurations you can use the `parse_combined_config_files`
+builtin. It accepts a list of file paths as the parameter and returns the parsed
+configuration as an list of Rego objects. When iterating over the combined input,
+the policy will return an list of results for each iteration of the policy over the
+input list. The example below shows a violation for deployments not matching a service
+spec selector from a combined manifest file input.
+
+**combine.rego**
+
+```rego
+violation[msg] {
+    some i
+    input[i].contents.kind == "Deployment"
+    deployment := input[i].contents
+    not service_selects_app(deployment.spec.selector.matchLabels.app)
+    msg := sprintf("Deployment '%v' has no matching service", [deployment.metadata.name])
+}
+service_selects_app(app) {
+    some i
+    input[i].contents.kind == "Service"
+    service := input[i].contents
+    service.spec.selector.app == app
+}
+```
+
+**combine_test.rego**
+
+```rego
+test_parse_combined_config_file {
+	count(violation) == 1 with input as parse_combined_config_files(["combine.yaml"])
+}
+```
+
+**combine.yaml**
+
+```yaml
```

Comment:
Can you reduce the size of this example to only include the deployment that matches the rule's test?